clang/test/SemaSYCL/sycl-dllimport-dllexport.cpp

```diff
@@ -0,0 +1,48 @@
+// RUN: %clang_cc1 -triple spir64-unknown-linux-sycldevice -fsycl-is-device -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECK
+
+// RUN: %clang_cc1 -triple spir64-unknown-windows-sycldevice -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECK
+
+// RUN: %clang_cc1 -triple spir64-unknown-windows-sycldevice -fsycl-is-device -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECKALL
+
+// RUN: %clang_cc1 -triple spir64-unknown-windows-sycldevice -fsycl-is-device -fms-extensions -fsyntax-only -verify  %s
```

Comment:
Probably, it's better to use two different tests instead of using 4 run commands, where the last one shares nothing with the first 3.
This will allows to remove vague "TEST" macro.

Reply:
I am not sure why you think this is the problem.  I'd like to have one test to test all my code.

But if you insist, I can change that.  Please let me know.